src/xmipp/bindings/python/envs_DLTK/xmipp_graph.yml

```diff
@@ -0,0 +1,13 @@
+name: xmipp_graph
+channels:
+  - anaconda
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.8
+  - scipy=1.10
+  - networkx=2.8
+  - cvxpy=1.3
+  - pip
+  - pip:
+      - numpy==1.23
```

Comment:
This version of NumPy is required? It could create conflicts with the one Scipion uses

Reply:
No, no specific version of numpy is required. I will remove the numpy requirement altogether

Reply:
Removed numpy requirement